portfolio/src/main/webapp/index.html

```diff
@@ -29,8 +29,6 @@
       <!-- About Me Section -->
       <section id="aboutme">
           <h1>About Me</h1>
-          <button onclick="sayHello()">Hello!</button>
-          <div id="hello-container"></div>
```

Comment:
Yup! For the POST comment form they instruct us to have the comments show upon load, so the button isn't needed anymore. I also moved the comments container div down where the Contact Me section is, since it seemed to fix better there instead.

Reply:
Sounds good!